src/Mod/Fem/App/FemSolverObject.cpp

```diff
@@ -1,5 +1,5 @@
 /***************************************************************************
- *   Copyright (c) 2013 J¨¹rgen Riegel (FreeCAD@juergen-riegel.net)        *
+ *   Copyright (c) 2013 Jürgen Riegel (FreeCAD@juergen-riegel.net)         *
```

Comment:
AFAIK code files should not have ü, but I'm not 100 % sure

Reply:
`grep -iR riegel` shows that `Jürgen` is correct.